railgun/internal/proxy/proxy.go

```diff
@@ -53,4 +54,10 @@ type Proxy interface {
 	// The delete action will asynchronously be converted to Kong DSL and applied to the Kong Admin API.
 	// A status will later be added to the object whether the configuration update succeeds or fails.
 	DeleteObject(obj client.Object) error
+
+	// DBMode indicate the database configuration in use by the running proxy (e.g. "off", "postgres")
+	DBMode() string
+
+	// Version indicates the version of the proxy
+	Version() semver.Version
```

Comment:
who uses these interface methods? Are they really needed?

Reply:
The manager uses them now.

Reply:
Not needed, removed in 36a1a1870a506f5250eca0c815c25535013cc922 :+1: